src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_3_layout.html.twig

```diff
@@ -232,6 +232,18 @@
     </div>
 {%- endblock radio_row %}
 
+{# Support #}
+
+{% block empty_row -%}
+    {%- set translation_domain = translation_domain ?: 'sf_form' -%}
+    {%- set attr = attr|merge({class: (attr.class|default('') ~ ' form-control')|trim}) %}
+    <div {{ block('widget_container_attributes') }}>
+        <p>
+            <em>{{- translation_domain is same as(false) ? empty_view : empty_view|trans({}, translation_domain) -}}</em>
```

Comment:
em isnt about styling? its about emphasis. and because there is no form, that one might expect, you put the emphasis on the presented message, probably giving reasons as for *why* there are no children.